taipy/gui/utils/get_module_name.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2022 Avaiga Private Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+# the License. You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+# an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+import inspect
+import sys
+import typing as t
+from types import FrameType, ModuleType
+
+
+def _get_module_name_from_frame(frame: FrameType):
+    return t.cast(ModuleType, inspect.getmodule(frame)).__name__
+
+
+def _get_module_name_from_imported_var(var_name: str, value: t.Any, sub_module_name: str) -> str:
+    module_list = sys.modules.keys()
```

Comment:
https://docs.python.org/3/library/sys.html#sys.modules
says we should use .copy() to iterate ....